libkatana_python_native/src/ImportData.cpp

```diff
@@ -39,7 +39,7 @@ katana::python::InitImportData(py::module& m) {
       [](py::array_t<PropertyGraph::Edge> edge_indices,
          py::array_t<PropertyGraph::Node> edge_destinations)
           -> Result<std::shared_ptr<PropertyGraph>> {
-        return KATANA_CHECKED(katana::PropertyGraph::Make(
+        return KATANA_CHECKED(katana::PropertyGraph::MakeEphemeral(
             TopologyFromCSR(edge_indices, edge_destinations)));
```

Comment:
A reasonable usage of this python function is to call it and then call `write` on the result to store it to a specific location. Will that work correctly? It relies on writing the RDG to a different location compared to where it started (in the tmp dir).

Reply:
I thought a lot about that use case when I was writing this so it should work correctly. But I need to check to see if we are testing it currently.

Reply:
I'm pretty sure I test it from Python.

Reply:
Could you point me in the right direction to short-circuit my search a little?

Reply:
https://github.com/KatanaGraph/katana/blob/95ad4cfcacf9b0d5b89b9dbff801c83c1339e179/python/test/test_convert_graph.py#L227

Through the power of searching for `write` then looking at the tests for importing.